crates/replace/src/lib.rs

```diff
@@ -349,13 +336,10 @@ impl<T: Config> Pallet<T> {
         );
 
         // check that that the total griefing collateral is sufficient
-        let required_collateral = ext::fee::get_replace_griefing_collateral::<T>(
```

Comment:
I can't comment on the line above, because you didn't change it, but this is no longer true

```
        // check that total-to-be-replaced is above the minimum. NOTE: this means that even
        // a request with amount=0 is valid, as long the _total_ is above DUST. This might
        // be the case if the vault just wants to increase the griefing collateral, for example.
   ```